spid-spring-integration/src/main/java/it/italia/developers/spid/integration/util/AuthenticationInfoExtractor.java

```diff
@@ -187,25 +187,25 @@ public AuthnRequest buildAuthenticationRequest(String assertionConsumerServiceUr
 		DateTime issueInstant = new DateTime();
 		AuthnRequestBuilder authRequestBuilder = new AuthnRequestBuilder();
 
-		AuthnRequest authRequest = authRequestBuilder.buildObject(SAML2_PROTOCOL, "AuthnRequest", "samlp");
-		authRequest.setIsPassive(Boolean.FALSE);
-		authRequest.setIssueInstant(issueInstant);
-		authRequest.setProtocolBinding(SAML2_POST_BINDING);
-		authRequest.setAssertionConsumerServiceURL(assertionConsumerServiceUrl);
-		authRequest.setAssertionConsumerServiceIndex(assertionConsumerServiceIndex);
-		authRequest.setIssuer(buildIssuer(issuerId));
-		authRequest.setNameIDPolicy(buildNameIDPolicy());
-		authRequest.setRequestedAuthnContext(buildRequestedAuthnContext());
-		authRequest.setID(id);
-		authRequest.setVersion(SAMLVersion.VERSION_20);
-
-		authRequest.setAttributeConsumingServiceIndex(1);
-		authRequest.setDestination(destination);
+		AuthnRequest authRequestResponse = authRequestBuilder.buildObject(SAML2_PROTOCOL, "AuthnRequest", "samlp");
```

Comment:
I'd prefer `authRequest` instead of `authRequestResponse`, do you agree?

Reply:
Yes